### ISSUE TITLE
Reindex (an probably other operations) is failing due to node's default HEADER size

### ISSUE THREAD

**Issue by RafaPolit** [huridocs/uwazi#2921]
Trying to make yarn reindex with node `v10.20.1` I have run into `HPE_HEADER_OVERFLOW` errors.

Node has a max header size limit of 8kb or 16kb (depending on version).

For the time being, I have been able to solve the reindex issue by launching the process individually, upping the max limit with:

```
$ node --max-http-header-size=80000 run.js ./database/reindex_elastic.js
```

We should look into this.

**Comment by RafaPolit** [comment]
Test downgrading Node to some known working version?  Perhaps Express?

**Comment by mfacar** [comment]
The same errors occur at indexing with the last version of Elasticsearch 7.7 even with Node v10.17.0, at returning to Elasticsearch 7.6 the errors disappear

**Comment by RafaPolit** [comment]
So, the culprit may very well be 7.7, as they got updated at the same time on my local machine.  I have not found a single report of those errors elsewhere for elastic 7.7 though.  Yes, lets see if we can make it work, or if we fix the version of Elastic to 7.6 until we have a better understanding.

**Comment by RafaPolit** [comment]
I have confirmed @mfacar 's findings: downgrading to 7.6.2 solves the issue.

For the time being, I have updated the README to reflect that we required a fix version of Elasticsearch.  For the time being, that version is v7.6.2.

The PR with the changes to the readme can be found in #2928
